src/c-writer.cc

```diff
@@ -388,11 +392,16 @@ std::string CWriter::Deref(const std::string& s) {
 // static
 char CWriter::MangleType(Type type) {
   switch (type) {
-    case Type::I32: return 'i';
-    case Type::I64: return 'j';
-    case Type::F32: return 'f';
-    case Type::F64: return 'd';
-    default: WABT_UNREACHABLE;
+    case Type::I32:
+      return 'i';
+    case Type::I64:
+      return 'j';
+    case Type::F32:
+      return 'f';
+    case Type::F64:
+      return 'd';
+    default:
+      WABT_UNREACHABLE;
```

Comment:
I agree that these change from compact to spread out form are potentially questionable.

Reply:
Added `AllowShortCaseLabelsOnASingleLine: true` to fix this.